### DISCUSSION TITLE
Kits from Tindie

### DISCUSSION THREAD

**Discussion by linuxplayground** [picocomputer/community#46] [Q&A]
Hey there,

I ordered a kit from the Agon Australia Tindie store a couple of weeks ago. 

My questions are:

1. Is this store officially linked or endorsed to or by this project in anyway?
2. If it is, is there any way I can get them to answer my emails via the Tindie message the store feature about shipping updates??  If it's not official then please let me know and others too.  I'm worried I've been scammed. 

Thanks 

Edit: I saw on another thread a similar conversation and it seems I should follow up there.

**Comment by rumbledethumps** [comment]
The only official Tindie store is "[Rumbledethumps](https://www.tindie.com/stores/rumbledethumps/)" but I don't ship international at this time. Being open source, other stores may appear, and that's ok, but I can't help with their orders.

I doubt you've been scammed. https://www.tindie.com/about/guarantee/

**Reply by linuxplayground** [reply]
Thanks. I understand. They have responded now.
